models/applications.ts

```diff
@@ -1,6 +1,56 @@
 import { application } from "../types/application";
 const firestore = require("../utils/firestore");
 const ApplicationsModel = firestore.collection("applicants");
+const { DOCUMENT_WRITE_SIZE: FIRESTORE_BATCH_OPERATIONS_LIMIT } = require("../constants/constants");
```

Comment:
Hi @vinayak-trivedi can we use a similar name here `DOCUMENT_WRITE_SIZE`, something related to `FIRESTORE_BATCH_OPERATIONS_LIMIT`  like `FIRESTORE_BATCH_WRITE_LIMIT`